protocols/v2/codec-sv2/examples/unencrypted.rs

```diff
@@ -0,0 +1,135 @@
+//! # Using Sv2 Codec Without Encryption
+//!
+//! This example demonstrates how to use the `codec-sv2` crate to encode and decode Sv2 frames
+//! without encryption over a TCP connection. It showcases how to:
+//!
+//! * Create an arbitrary custom message type (`CustomMessage`) for encoding and decoding.
+//! * Encode the message into a Sv2 frame.
+//! * Send the encoded frame over a TCP connection.
+//! * Decode the Sv2 frame on the receiving side and extract the original message.
+//!
+//! ## Run
+//!
+//! ```
+//! cargo run --example unencrypted
+//! ```
```

Comment:
```suggestion
// # Using Sv2 Codec Without Encryption
//
// This example demonstrates how to use the `codec-sv2` crate to encode and decode Sv2 frames
// without encryption over a TCP connection. It showcases how to:
//
// * Create an arbitrary custom message type (`CustomMessage`) for encoding and decoding.
// * Encode the message into a Sv2 frame.
// * Send the encoded frame over a TCP connection.
// * Decode the Sv2 frame on the receiving side and extract the original message.
//
// ## Run
//
// ```
// cargo run --example unencrypted
// ```
```